register.php

```diff
@@ -0,0 +1,103 @@
+<?php
+// Include the connection file
+include 'connection.php';
+// Check the button Register and INSERT the data in table users from the webpage form
+if (isset($_POST['register'])) {
```

Comment:
we have 0 sanitization here and we trust the end user, this one should be updated